src/PhpSpreadsheet/Worksheet/RowCellIterator.php

```diff
@@ -173,6 +173,21 @@ public function getCurrentColumnIndex()
     {
         return $this->currentColumnIndex;
     }
+    
+     /**
+     * Return the current Cell by Column position.
+     * Row Iterator already knows the current $rowIndex
+     * @param string $column The column address for the cell
+     * Edited Shahroz BUTT
+     * Email : shahroz.butt@yahoo.com
+     *
+     * @return Cell
+     */
+    public function getCellByColumn($column)
+    {
+        $column = Coordinate::columnIndexFromString($column);
```

Comment:
This functionality can be achieved using `$myRowCellIterator->seek('B')->current();` which has the added advantage of validating that the input column is within the specified range; though it also has the side-effect of updating the iterator index which may or may not be the required behaviour.

If the user wanted a column that wasn't within the rowCellIterator range of columns, then the functionality should really be in the rowIterator, rather than in the rowCellIterator. That might well be a better option anyway, because it gives direct access to cells at a higher-level, and leaves the cellIterator as a pure iterator.